packages/code-du-travail-frontend/src/outils/IndemniteLicenciement/index.tsx

```diff
@@ -30,8 +35,9 @@ function CalculateurIndemnite({ icon, title }) {
         {async (value) => {
           if (value) {
             const module = await import(`./ccn/${value}`);
-            const steps = module.steps.filter(({ condition = () => true }) =>
-              condition(values)
+            const steps = module.steps.filter(
+              // eslint-disable-next-line @typescript-eslint/no-unused-vars
+              ({ condition = (values) => true }) => condition(values)
```

Comment:
Le commentaire m'a mis la puce à l'oreille. J'ai l'impression que l'on peut remplacer par 
```suggestion
              () => true
```

Ici, on a condition qui est une fonction qui retourne true tout le temps. Donc le condition(values) retourne toujours `true` actuellement.

On peut même virer le filter car il ne filtre rien d'ailleurs 🤔